### PR TITLE
Add new master link for newly approved apps

### PR DESCRIPTION
**Pull request by sgchoi5**

### CODE REVIEW

File: docs/application/dotnet/samples/tizen-samples.md [hunk 1]
```diff
@@ -254,6 +251,7 @@
 			</td>
 			<td>
 			<p><strong>FirstScreen</strong></p>
+                        <p>FirstScreen is a sample application that demonstrates how to launch the apps on the TV.</p>
```
Comment (mijin85cho) [inline docs/application/dotnet/samples/tizen-samples.md]: Review result from annie-samsung
```
If FirstScreen sample app lets you to launch only one app on the TV, then please change the sentence to:

FirstScreen is a sample application that demonstrates how to launch an app on the TV.
```

Or, if current sentence is correct, please let me know.

Comment (sgchoi5) [reply docs/application/dotnet/samples/tizen-samples.md]: Not for only one app, FirstScreen can launch several apps

Comment (mijin85cho) [reply docs/application/dotnet/samples/tizen-samples.md]: Thanks for the check.

**Comment by mijin80lee** [comment]
TD Sync Completed.

**Pull request merged**